MM/5.8.0/config/media_manager_5/fields/custom_quality_color_spaces.dcl

```diff
@@ -5,10 +5,17 @@ resource configservice_combo_config_field_option custom_quality_color_spaces__cm
     is_default_selected = true
 }
 
-resource configservice_combo_config_field_option custom_quality_color_spaces__grayscale {
+resource configservice_combo_config_field_option custom_quality_color_spaces__gray {
     configservice_field_id = resource.configservice_multi_combo_config_field.custom_quality_color_spaces.id
-    value = 'Grayscale'
-    title = 'Grayscale'
+    value = 'gray'
+    title = 'Gray'
+    is_default_selected = true
```

Comment:
Just change the `value` here, as that is what is send back to DC.

Reply:
done